### DISCUSSION TITLE
Possible to train a model?

### DISCUSSION THREAD

**Discussion by misslollipopofficial** [rupeshs/fastsdcpu#131] [General]
Sorry I'm pretty new to stable diffision etc maybe I have not the knowledge to make what I want. I already have created some images but I want to fine tune these images. Like is that whats training a model is about? Is that possible?

**Comment by monstruosoft** [comment]
You probably don't want to train a model, you may just need to find an already existing model with the image style you want.
Try searching models on [huggingface](https://huggingface.co/).
